docs/troubleshooting/installation.md

```diff
@@ -25,13 +25,48 @@ default via 10.10.0.10 dev harvester-mgmt proto dhcp        <-- Does a default r
 
 - Check that your DHCP server offers a default route option. Attaching content from `/run/cos/target/rke2.log` is helpful too.
 
+## Modifying cluster token on agent nodes
+
+For agent nodes to join a cluster, they must have a cluster token identical to the one set up by the
+server node. To see whether nodes are unable to join a cluster is caused by unmatched token,
+SSH into the agent nodes and check the log of `rancherd` service:
+
+```bash
+$ sudo journalctl -b -u rancherd
+```
+
+If you see repeated log messages like this:
+
+```
+...
+msg="Bootstrapping Rancher (master-head/v1.21.5+rke2r1)"
+msg="failed to bootstrap system, will retry: generating plan: insecure cacerts download from https://192.168.122.115:8443/cacerts: Get \"https://192.168.122.115:8443/cacerts\": EOF"
+...
+```
+
+You may have an incorrect token. To update it, edit the line `token: <YourToken>`
+of `/etc/rancher/rancherd/config.yaml`. For example, if the correct token is
+`ThisIsTheCorrectOne`:
+
+```yaml
+...
+token: 'ThisIsTheCorrectOne'
+...
+```
+
+If you are not sure what the current token is, SSH into the server node and enter the
+following command to show it:
+```bash
+$ sudo yq e .token /etc/rancher/rancherd/config.yaml
+```
```

Comment:
```suggestion
!!! note
    To see what is the current cluster token value, log in your server node (i.e. with SSH) and look in the file `/etc/rancher/rancherd/config.yaml`. For example, you can run the following command to only display the token's value:
    ```bash
    $ sudo yq -e .token /etc/rancher/rancherd/config.yaml
    ```
```

Reply:
Rephrased. However, for the `yq` command the flag `-e` has a different meaning from `e`. I've replaced it with `eval` to increase readability:
```bash
$ sudo yq eval .token /etc/rancher/rancherd/config.yaml
```